internal/pkg/template/templates/workloads/partials/cf/instancerole.yml

```diff
@@ -19,7 +19,10 @@ InstanceRole:
       Statement:
         - Effect: Allow
           Principal:
-            Service: tasks.apprunner.amazonaws.com
+            Service:
+              - tasks.apprunner.amazonaws.com
+              - !Sub '${AWS::Region}.beta.credentialvendingserviceproxy.bullet.aws.internal' # TODO remove on release
```

Comment:
nit: my suggestion is to keep the internal endpoints + VPC connector change in a separate commit that you can always cherry-pick while locally testing but keeping the PR without these changes

Reply:
i'll leave it for now just since i'm merging into a branch - but good idea for future 😊